guest-os-info.md

```diff
@@ -0,0 +1,53 @@
+## Guest Operating System Information
+
+Guest operating system identity for the Virtual Machine will be provided by the label ``os-short-id`` :
+
+```
+metadata:
+  name: myvm
+  labels:
+    os-short-id: win2k12r2
+```
+
+The ``os-short-id`` label is based on the short OS identifier from [libosinfo](https://libosinfo.org/).
```

Comment:
Do we really want to expose `kubevirt.io/os-id` too? I would prefer not to, since the not-short is is a mix of a http url + an identifier. Also how would we handle conflicting identifiers then?

Consider renaming the label to `kubevirt.io/os`.

Reply:
We want to expose only ``kubevirt.io/os-short-id``   (or ``kubevirt.io/os`` if we decide to rename it)
using only the os-short-id we expect that other properties could be extracted from libosinfo if one desire to do so. 

If we use only the os-short-id  it should be unique , or do you refer to other type of conflicts ?

Reply:
+1 on exposing `kubevirt.io/os-short-id` only